frontend/src/Components/Endpoint/Endpoint.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "~plaid-threads/scss/variables";
+
+.endpointContainer {
+  display: grid;
+  grid-template-columns: 15% 57% 28%;
+  width: 100%;
+  border: 1px solid $black200;
+  border-bottom: 0px;
+}
+
+.post {
+  margin: 2 * $unit 0px 2 * $unit 3 * $unit;
+  font-size: 1.4rem;
+}
+.endpointContents {
+  margin: 3 * $unit 4 * $unit;
+  font-size: 1.2rem;
+}
+
+.endpointHeader {
+  margin-bottom: $unit;
+}
+
+.endpointName {
+  font-weight: bold;
+  font-size: 2 * $unit;
+  padding-right: $unit;
+}
+
+.schema {
+  font-size: 2 * $unit;
+  font-family: "Inconsolata";
+  letter-spacing: -0.24px;
+}
+.endpointDescription {
+  font-size: 1.2rem;
+  line-height: normal;
+}
+
+.sendRequest {
+  margin: 2 * $unit;
+  width: 70%;
+  margin-left: 3 * $unit;
```

Comment:
is this width and margin doing anything? its inside a css grid. In a css grid you can use `grid-column-gap` to define gaps between columns instead of using margins and widths

Reply:
the width of the button itself is too big, based on threads css.  without width, it exceeds the size of the css grid column.